### PR TITLE
Do stuff (some stuff is rejected, some is not)

### PR DESCRIPTION
**Pull request by Time-dragon**
Resolves nothing.

### Changes

Stuff

### Reason for changes

Why not?

### Tests

### CODE REVIEW

File: addons/dark-www/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Website dark mode",
-  "description": "Dark theme for the Scratch website, both for 2.0 and 3.0 pages.",
+  "name": "Main site dark mode",
```
Comment (Shluffy) [inline addons/dark-www/addon.json:2]: website sounds better

File: addons/dark-www/addon.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "description": "Dark theme for the Scratch website, for 2.0-styled and 3.0-styled pages.",
```
Comment (Shluffy) [inline addons/dark-www/addon.json:3]: it could be "both for 2.0 and 3.0 **styled** pages."

File: addons/discuss-button/addon.json [hunk 3]
```diff
@@ -34,5 +34,5 @@
   ],
   "versionAdded": "1.0.0",
   "tags": ["community", "forums"],
-  "enabledByDefault": false
+  "enabledByDefault": true
```
Comment (Shluffy) [inline addons/discuss-button/addon.json:37]: normally stuff are false rather then true. This should probably be false

File: addons/blocks2image/addon.json [hunk 4]
```diff
@@ -15,5 +15,5 @@
     }
   ],
   "versionAdded": "1.13.0",
-  "enabledByDefault": false
+  "enabledByDefault": true
```
Comment (Shluffy) [inline addons/blocks2image/addon.json:18]: again, should probably be false

File: addons/blocks2image/addon.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Save blocks as image",
   "description": "Right click the editor workspace to export blocks as SVG/PNG images.",
-  "tags": ["editor", "codeEditor"],
+  "tags": ["editor", "codeEditor" "recommended"],
```
Comment (Shluffy) [inline addons/blocks2image/addon.json:4]: probably shouldn't be recommened

File: README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <img src="https://raw.githubusercontent.com/ScratchAddons/ScratchAddons/master/images/icon.svg" alt="Scratch Addons logo" align="right" width="128px"></img>
 # Welcome to Scratch Addons' repository!
-
+## Badges
```
Comment (Shluffy) [inline README.md:3]: not needed

File: README.md [hunk 7]
```diff
@@ -16,6 +16,7 @@
 [![](https://img.shields.io/badge/discuss-on_github-181717.svg?style=flat-square)](https://github.com/ScratchAddons/ScratchAddons/discussions)
 [![](https://img.shields.io/badge/chat-on_discord-7289da.svg?style=flat-square)](https://discord.gg/R5NBqwMjNc)
 [![](https://img.shields.io/badge/website-scratchaddons.com-ff7b26.svg?style=flat-square)](https://scratchaddons.com)
+[![Join the chat at https://gitter.im/AddonChat/community](https://badges.gitter.im/AddonChat/community.svg)](https://gitter.im/AddonChat/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (Shluffy) [inline README.md]: is this gitter thing by sa or by you?

File: addons/slow-motion/userscript.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
+export default async function ({ addon, global, console }) {
```
Comment (Shluffy) [inline addons/slow-motion/userscript.js:1]: this looks the same as the 60 fps one

Comment (Shluffy) [reply addons/slow-motion/userscript.js:1]: the whole addon code

**Comment by Shluffy** [comment]
I don't think you are supposed to make prs using the master branch

**Comment by GrahamSH-LLK** [comment]
This entire pull request is useless

File: addons/60fps/addon.json [hunk 9]
```diff
@@ -17,6 +17,14 @@
...
       "link": "https://scratch.mit.edu/users/TheColaber"
     }
+    {
+      "name": "GarboMuffin",
+      "link": "https://scratch.mit.edu/users/GarboMuffin"
+    }
+    {
+      "name": "TurboWarp by GarboMuffin",
+      "link": "https://turbowarp.org"
+    }
```
Comment (FunctionalMetatable) [inline addons/60fps/addon.json:27]: Not needed. The credits only need people, not other things

File: addons/60fps/addon.json [hunk 10]
```diff
@@ -17,6 +17,14 @@
       "name": "TheColaber",
       "link": "https://scratch.mit.edu/users/TheColaber"
     }
+    {
```
Comment (FunctionalMetatable) [inline addons/60fps/addon.json:20]: add a comma after line 19 or json will be malformed

File: addons/debugger/userscript.js [hunk 11]
```diff
@@ -97,7 +97,7 @@ export default async function ({ addon, global, console, msg }) {
     myFlash.block = block;
 
     function _flash() {
-      myFlash.block.svgPath_.style.fill = flashOn ? "#ffff80" : myFlash.colour;
+      myFlash.block.svgPath_.style.fill = flashOn ? "#ff7b26" : myFlash.colour;
```
Comment (FunctionalMetatable) [inline addons/debugger/userscript.js:100]: Why?

File: addons/discuss-button/addon.json (hunk 3 above)
Comment (FunctionalMetatable) [reply addons/discuss-button/addon.json:37]: Addons that are normally enabled by default have to go through approval by @WorldLanguages I think..

File: addons/editor-dark-mode/addon.json [hunk 12]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Editor dark mode and customizable colors",
+  "name": "Dark mode with custom editor colors",
```
Comment (FunctionalMetatable) [inline addons/editor-dark-mode/addon.json:2]: Editor dark mode would be a better name for this

File: addons/slow-motion/userscript.js (hunk 8 above)
Comment (FunctionalMetatable) [reply addons/slow-motion/userscript.js:1]: It is literally the 60fps code.

File: addons/slow-motion/userscript.js [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+      if (this._steppingInterval) return;
+      let interval = 1000 / global_fps;
+      this.currentStepTime = interval;
+      this._steppingInterval = setInterval(() => {
+        this._step();
+      }, interval);
+      this.emit("RUNTIME_STARTED");
+    };
+  }
+}
```
Comment (FunctionalMetatable) [inline addons/slow-motion/userscript.js:55]: Needs to be in a different PR, and again, the same as the 60fps addon

**Comment by FunctionalMetatable** [comment]
oops i made the review request after it was closed lol

File: addons/slow-motion/userscript.js (hunk 13 above)
Comment (Time-dragon) [reply addons/slow-motion/userscript.js:55]: @FunctionalMetatable You can help with userscript.

File: addons/60fps/addon.json (hunk 9 above)
Comment (Time-dragon) [reply addons/60fps/addon.json:27]: TurboWarp should be credited though

File: addons/blocks2image/addon.json (hunk 5 above)
Comment (Time-dragon) [reply addons/blocks2image/addon.json:4]: Just want it widespread.

File: addons/debugger/userscript.js (hunk 11 above)
Comment (Time-dragon) [reply addons/debugger/userscript.js:100]: So that it is Scratch Addons-styled not Scratch styled

Comment (mxmou) [reply addons/debugger/userscript.js:100]: > So that it is Scratch Addons-styled not Scratch styled

It should be Scratch-styled, because the addon runs in the Scratch editor.
